FAIL
Kernel panic in run: PANIC at ../../threads/synch.c:223 in lock_release(): assertion `lock_held_by_current_thread (lock)' failed.
Call stack: 0x800421342b 0x8004209e2e 0x8004217ddf 0x800420727a
Translation of call stack:
0x000000800421342b: debug_panic (lib/kernel/debug.c:32)
0x0000008004209e2e: lock_release (threads/synch.c:225)
0x0000008004217ddf: b_thread_func (tests/threads/priority-donate-multiple.c:76)
0x000000800420727a: kernel_thread (threads/thread.c:423)
